modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/css/Actions.scss

```diff
@@ -1,5 +1,15 @@
 @import 'atlas-variables';
 
+.icons-list ul li {
```

Comment:
`icons-list` sounds a bit generic, especially for a CSS class that will end up in HTML body root. It'll be quite possible that some customer's CSS affects this. Can we change this to something specific, like `dsm-actions-icon-selection-modal`.

Reply:
Good point, I changed it